### PR TITLE
Lesson 11

### PR DESCRIPTION
**Pull request by vankampenc**

### CODE REVIEW

**Comment by vankampenc** [comment]
Added to the incorrect repo. Closing this PR

File: CSS/index.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    font-size: 20px;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+header {
+    background-color: black;
+}
+
+.sticky {
```
Comment (mannsn) [inline CSS/index.css:51]: To make it not scroll over the sections, for the section headers you should add something like: 

#About{
    scroll-margin-top: 5em;
}

That should help with the scrolling issues.

File: CSS/index.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.boldFont {
+    font-weight: bolder;
+}
+
+nav {
+    display: flex;
+    gap: 30px;
+    justify-content: flex-end;
```
Comment (mannsn) [inline CSS/index.css:33]: Note with flex end on large displays everything shifts to right.  I assume that is what you wanted though.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                <i class="fa fa-linkedin-square"></i>
+                </a>
+                
+                <a title="link to Christopher Van Kampen's Github" href="https://github.com/vankampenc" target="_blank">
+                <i class="fa-brands fa-github"></i>
+                </a>
+            </div>
+    </section>
+
+    <section id="Skills">
```
Comment (mannsn) [inline index.html:104]: Should go before connect

**Review by mannsn — COMMENTED** [review]
Very nice job!   Put comments in file, but you just need to add scroll-margin-top to your section headers to avoid scrolling over the content. Nice linking to resume and to other elements.   Nav bar and connect bar look good.   Would make connect at bottom though.

File: CSS/index.css (hunk 1 above)
Comment (vankampenc) [reply CSS/index.css:51]: Updated

File: CSS/index.css (hunk 2 above)
Comment (vankampenc) [reply CSS/index.css:33]: That is correct. I did want it to the right side of the page :)

File: index.html (hunk 3 above)
Comment (vankampenc) [reply index.html:104]: Updated

**Pull request closed without merging**